src/v_trans.c

```diff
@@ -174,11 +195,17 @@ byte V_Colorize (byte *playpal, int cr, byte source)
     rgb.y = playpal[3 * source + 1] / 255.;
     rgb.z = playpal[3 * source + 2] / 255.;
 
+    fesetround(FE_DOWNWARD); // lrint
+
     rgb_to_hsv(&rgb, &hsv);
 
-    if (cr == CR_GRAY || cr == CR_BLACK || cr == CR_WHITE)
+    if (cr == CR_BRIGHT)
+    {
+        hsv.z *= 1.4;
+    }
```

Comment:
This will need to have its value capped. Also, how about custom font graphics with a color that is already close to full bright?

Reply:
There won't be a single translation which fits all use cases. Highlighting won't work for very bright fonts, shading the non-selected entries won't work for very dark fonts. Translating to orange will look crap for some other fonts, but at least it's Boom heritage. Dsda uses the "brick" table, I think, but I don't like it either.

Reply:
I think highlighting is the safest option. We still have `<`, `>` indicators if all else fails.

Reply:
> What is the cap? I don't know much about this code.

I think it must not exceed `1.0`.

> I think highlighting is the safest option. We still have `<`, `>` indicators if all else fails.

Hm, alright, let's see how it works out.

Reply:
https://github.com/fabiangreffrath/woof/assets/5077629/9cb37914-075f-402f-8f52-4b3888e1e701

Reply:
Oh, that's nice, thank you!

Reply:
How does this look with a gray themed font as in BTSX (forgot if E1 or E2)?

Reply:
BTSX looks good:

https://github.com/fabiangreffrath/woof/assets/5077629/41a20621-cde9-488f-b31f-1dc50b183541

I've tested it with my collection and so far it looks good everywhere.

Reply:
Could you check if it works if you set `hsv.z = 1.0f`?